android/src/main/AndroidManifest.xml

```diff
@@ -0,0 +1,11 @@
+<manifest xmlns:android="http://schemas.android.com/apk/res/android"
+    package="com.novoda.android">
+
+    <application android:allowBackup="true"
+        android:label="@string/app_name"
+        android:supportsRtl="true"
```

Comment:
but I was thinking if our library supprts RTL (which I believe it does) we should have it set to `true`. 

We are declaring our library supports RTL